xml/favicons.xml

```diff
@@ -0,0 +1,1696 @@
+    <description>HashiCorp Vault</description>
+    <example>c02e22ca67f83a0fb6f2fd265074910a</example>
+    <example>68e1a9c89026b0efeddf718a48c282a5</example>
+    <param pos="0" name="service.vendor" value="HashiCorp"/>
+    <param pos="0" name="service.product" value="Vault"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:hashicorp:vault:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^74120b5bbc7be340887466ff6cfe66c6$">
+    <description>Apple CUPS 1.3.7</description>
+    <example>74120b5bbc7be340887466ff6cfe66c6</example>
+    <param pos="0" name="service.vendor" value="Apple"/>
+    <param pos="0" name="service.product" value="CUPS"/>
+    <param pos="0" name="service.family" value="CUPS"/>
+    <param pos="0" name="service.version" value="1.3.7"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:apple:cups:1.3.7"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2c2bd163799c49e25bc7ac0fe97f2449$">
+    <description>Apple CUPS 1.x</description>
+    <example>2c2bd163799c49e25bc7ac0fe97f2449</example>
+    <param pos="0" name="service.vendor" value="Apple"/>
+    <param pos="0" name="service.product" value="CUPS"/>
+    <param pos="0" name="service.family" value="CUPS"/>
+    <param pos="0" name="service.version" value="1.x"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:apple:cups:1.x"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(?:448fa67ece2a35d57e8d19808d350e9b|0c9183bd23ed47288c733499fce4498c|c4302a71aae96e158c3057c66184abbd)$">
+    <description>Apple CUPS 2.x</description>
+    <example>448fa67ece2a35d57e8d19808d350e9b</example>
+    <example>0c9183bd23ed47288c733499fce4498c</example>
+    <example>c4302a71aae96e158c3057c66184abbd</example>
+    <param pos="0" name="service.vendor" value="Apple"/>
+    <param pos="0" name="service.product" value="CUPS"/>
+    <param pos="0" name="service.family" value="CUPS"/>
+    <param pos="0" name="service.version" value="2.x"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:apple:cups:2.x"/>
+  </fingerprint>
+
+  <fingerprint pattern="^cee18e28257988b40028043e65a6c2a3$">
+    <description>TeamCity</description>
+    <example>cee18e28257988b40028043e65a6c2a3</example>
+    <param pos="0" name="service.vendor" value="JetBrains"/>
+    <param pos="0" name="service.product" value="TeamCity"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:jetbrains:teamcity:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^e48c482f8f5a8e5a6249b21a39f911e7$"><description>Cockroach DB Console</description><example>e48c482f8f5a8e5a6249b21a39f911e7</example><param pos="0" name="service.vendor" value="Cockroach Labs"/><param pos="0" name="service.product" value="CockroachDB"/>&gt;
```

Comment:
formatting

Reply:
thanks! I wonder what did that, maybe a hanging > character and the being autoformatted by the cpe script